Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN hack/build.sh
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 RUN INSTALL_PKGS=" \
       which tar wget hostname sysvinit-tools util-linux \
-      socat tree findutils lsof bind-utils \
+      socat tree findutils lsof bind-utils fuse-overlayfs\
```

Comment:
The base image is derived from RHEL 7.6 at the moment.  Is fuse-overlayfs available there?

Reply:
Not sure, but I believe so.
@giuseppe Do you know?

Reply:
AFAIK, fuse-overlayfs is not available on RHEL 7.6, since it is not usable there (at least not from a user namespace)

Reply:
@lsm5 is fuse-overlayfs available for RHEL-7.6?  Do we plan to do it?

Reply:
@nalind I believe we have a tech debt card to base this on UBI, as we do for other OpenShift images

Reply:
Where is registry.svc.ci.openshift.org/openshift/origin-v4.0:base defined?  Can we update this to RHEL8/UBI8?

Reply:
Actually - this base image is replaced by CI, see https://github.com/openshift/release/blob/master/ci-operator/config/openshift/builder/openshift-builder-master.yaml#L2-L5

I believe this is in fact UBI based already (not sure if it's 7 or 8 - @bparees @stevekuznetsov can we verify this?)

Reply:
`Dockerfile-dev` is intended for contributor use - currently it's our public CentOS 7 image.

If "downstream" is building on top of UBI8, then I don't see an issue updating this to UBI8 as well.

Reply:
it's ubi7 currently, as @adambkaplan notes we're based on a common openshift base so when it moves, we'll move automatically.

```
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.1.3 --pullspecs | grep builder
  docker-builder                                quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:883dbe5905ae08f05598fc82cd5e470f1f2c2aaac86e9976351fda3cdb924a51

$ docker run -it --entrypoint=/bin/sh quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:883dbe5905ae08f05598fc82cd5e470f1f2c2aaac86e9976351fda3cdb924a51 
Unable to find image 'quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:883dbe5905ae08f05598fc82cd5e470f1f2c2aaac86e9976351fda3cdb924a51' locally
sha256:883dbe5905ae08f05598fc82cd5e470f1f2c2aaac86e9976351fda3cdb924a51: Pulling from openshift-release-dev/ocp-v4.0-art-dev
18f0f6de4601: Already exists 
db1d55616933: Already exists 
25d3230a0c3e: Already exists 
f7a985d90b92: Pull complete 
Digest: sha256:883dbe5905ae08f05598fc82cd5e470f1f2c2aaac86e9976351fda3cdb924a51
Status: Downloaded newer image for quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:883dbe5905ae08f05598fc82cd5e470f1f2c2aaac86e9976351fda3cdb924a51

sh-4.2# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.6 (Maipo)

sh-4.2# uname -a
Linux ea33ab12ba9d 5.1.6-300.fc30.x86_64 #1 SMP Fri May 31 17:43:23 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

sh-4.2# cd /etc/yum.repos.d/
sh-4.2# ls
redhat.repo  ubi.repo
```

Reply:
Please use Podman for your examples :^)

Reply:
Also could you see if you can install fuse-overlayfs, I don't think you can on RHEL7 based images.

Reply:
I don't suppose you'd believe me if i told you i had it aliased.....

```
# yum install fuse-overlayfs
Loaded plugins: ovl, product-id, search-disabled-repos, subscription-manager
This system is not registered with an entitlement server. You can use subscription-manager to register.
ubi-7                                                              | 3.8 kB  00:00:00     
ubi-7-optional                                                     | 3.7 kB  00:00:00     
ubi-7-rhah                                                         | 3.7 kB  00:00:00     
ubi-7-rhscl                                                        | 3.8 kB  00:00:00     
(1/12): ubi-7/x86_64/updateinfo                                    |   92 B  00:00:00     
(2/12): ubi-7/x86_64/group                                         |  124 B  00:00:00     
(3/12): ubi-7/x86_64/primary_db                                    | 675 kB  00:00:00     
(4/12): ubi-7-optional/x86_64/group                                |  124 B  00:00:00     
(5/12): ubi-7-optional/x86_64/updateinfo                           |   92 B  00:00:00     
(6/12): ubi-7-optional/x86_64/primary_db                           |  13 kB  00:00:00     
(7/12): ubi-7-rhah/x86_64/group                                    |  124 B  00:00:00     
(8/12): ubi-7-rhah/x86_64/updateinfo                               |   92 B  00:00:00     
(9/12): ubi-7-rhah/x86_64/primary_db                               | 4.0 kB  00:00:00     
(10/12): ubi-7-rhscl/x86_64/group                                  |  124 B  00:00:00     
(11/12): ubi-7-rhscl/x86_64/updateinfo                             |   92 B  00:00:00     
(12/12): ubi-7-rhscl/x86_64/primary_db                             | 158 kB  00:00:00     
No package fuse-overlayfs available.
```

Reply:
Thanks I could not pull that image myself.  Looks like I will need to wait for the upgrade to add fuseoverlayfs to that Dockerfile.